core/jobs/io/firebase_io_test.py

```
@@ -0,0 +1,226 @@
+        )
+        self.firebase_sdk_stub.create_user(
+            uid='uid_b', email='b@b.com', disabled=False
+        )
+        self.firebase_sdk_stub.create_user(
+            uid='uid_c', email='c@c.com', disabled=True
+        )
+
+        self.assert_pcoll_equal(
+            self.pipeline | firebase_io.GetStrongRecords(),
+            [
+                firebase_adapters.StrongRecord('uid_a', 'a@a.com', False),
```

Ruff (0.15.0)

[error] Boolean positional value in function call

(FBT003)